docs/CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+# lockbox-extension code owners
+# https://help.github.com/articles/about-codeowners/
+
+# tests should also be reviewed by @m8ttyB
+/test/    @jimporter @m8ttyB
+
+# docs are shepherded by @devinreams
+/docs/    @jimporter @devinreams
+
+# otherwise, everything is to be reviewed by @jimporter
+*         @jimporter
```

Comment:
Should also possibly ping @flodolo for any en-US/*.ftl l10n string changes:

```
src/webextension/locales/en-US/*.ftl @flodolo
```

**NOTE:** Assumes that the #87 PR lands which rearranges *.ftl file locations.

Reply:
Good catch @pdehaan! I saw this on the Test Pilot checklist the other day and appreciate you looking out. 👍